assets/sass/patterns/molecules/article-section.scss

```diff
@@ -373,6 +373,60 @@ $arrow-size: 5;
   }
 }
 
+.section-wrapper {
+  @media only all and (min-width: #{get-rem-from-px($bkpt-site--small)}em) {
+    display: grid;
```

Comment:
As seeing as we are starting to use this in a few places we could make a mixin with lines 378,379 and 380

Reply:
Good idea, thanks, Dave! The mixin grid is created in order to reuse the logic for grid display. The files info-bars.scss and content-header-journal.scss are not related to this PR but they can use the mixin too.